cmd/daemon/main_test.go

```diff
@@ -4,69 +4,57 @@ import (
 	"context"
 	"flag"
 	"fmt"
-	"strconv"
 	"testing"
 
 	"github.com/stretchr/testify/require"
-	"github.com/urfave/cli"
+	"github.com/urfave/cli/v2"
 )
 
-func newTestContext(t *testing.T, description string, flags []string, values []interface{}) *cli.Context {
-	require.Equal(t, len(flags), len(values))
-
+func newTestContext(t *testing.T, description string, flags map[string]any) *cli.Context {
```

Comment:
I moved us from using 2 separate arrays with synced indices to a map triggering a minor rewrite below.